inscriptions/ISic1645.xml

```diff
@@ -169,51 +174,30 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="EDR" n="EDR073713" xml:lang="la">
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#VM">
                 <ab>
-                    <lb n="1"/>
-                    <supplied reason="lost">
-                        <g type="aplha"/>
-                    </supplied>
-                    <g type="christogramma"/>
-                    <g type="omega"/>
-                    Auxentius His<lb n="2" break="no"/>panus patria
-                    <expan>
-                        <abbr>ep</abbr>
-                        <ex>iscopus</ex>
-                    </expan>
-                    Rotdon<lb n="3"/>iacet
-                    <choice>
-                        <sic>huc</sic>
-                        <corr>hic</corr>
-                    </choice>
-                    et
-                    <choice>
-                        <sic>aiuro</sic>
-                        <corr>adiuro</corr>
-                    </choice>
-                    vos qui legi
-                    <lb n="4" break="no"/>
-                    <choice>
-                        <sic>tes</sic>
-                        <corr>legitis</corr>
-                    </choice>
-                    <gap reason="lost" extent="unknown" unit="character"/>
-                    petite q
-                    <supplied reason="lost">uo</supplied>
-                    <lb n="5" break="no"/>d acce<supplied reason="lost">ptum</supplied>.
+                    <lb n="1"/><supplied reason="lost"><g type="aplha"/></supplied> <g type="staurogram">&#65952;</g> <g type="omega"/> Auxentius His
+                    <lb n="2" break="no"/>panus patria <expan><abbr>ep</abbr><ex>iscopus</ex></expan> Rotdon
+                    <lb n="3"/>iacet <choice><corr>hic</corr><sic>huc</sic></choice> et a<supplied reason="omitted">d</supplied>iuro vos qui legi
+                    <lb n="4" break="no"/><choice><corr>tis</corr><sic>tes</sic></choice> <gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="4a"/><gap reason="lost" extent="unknown" unit="character"/>
```

Comment:
So, here I should use too: `<gap reason="lost" extent="unknown" unit="line"/>`?

Reply:
If the edition is confident that it's just one line, it will still be `quantity="1" unit="line"` but the `<lb>` with have number 5 rather than 4a, assuming reconstruction is theoretically possible.

Reply:
This looks like a vacat, though, if it's in the middle of the text.